contracts/truefi3/interface/IDebtInstrument.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
```

Comment:
Let's use solidity >0.8 in truefi 3

Reply:
which version should we use specifically?